include/triton/core/tritonbackend.h

```diff
@@ -95,7 +95,7 @@ struct TRITONBACKEND_Batcher;
 ///   }
 ///
 #define TRITONBACKEND_API_VERSION_MAJOR 1
-#define TRITONBACKEND_API_VERSION_MINOR 19
+#define TRITONBACKEND_API_VERSION_MINOR 20
```

Comment:
There isn't any backend API change is there?

Reply:
Yes, no change in the interface, so not big enough of a change to update the version.